package.json

```diff
@@ -3,61 +3,45 @@
   "version": "0.0.3",
   "scripts": {
     "interactive": "./scripts/interactive.js",
-    "nightwatch": "nightwatch -c ./conf/dev.js",
+    "nightwatch": "nightwatch -c ./build/conf/dev.js",
     "nightwatch-chrome": "nightwatch -c ./conf/dev.js --env CHROME_LATEST",
     "nightwatch-ff": "nightwatch -c ./conf/dev.js --env FIREFOX_LATEST",
     "browserstack": "./scripts/local.runner.js -c ./conf/browserstack-default.js",
     "browserstack-all": "./scripts/local.runner.js -c ./conf/browserstack-chrome.js; ./scripts/local.runner.js -c ./conf/browserstack-firefox.js",
     "deploy": "git push && git push --tags",
-    "flow": "flow"
+    "clean": "rimraf ./build",
+    "type-check": "tsc -p ./tsconfig.json",
+    "prebuild": "npm run clean",
+    "build": "npm run type-check",
```

Comment:
is it possible to have a type-check that doesn't run a transpile? also I think the build is gonna have to also copy the static assets (namely the tsv test files) to the build directory too

Reply:
I don’t think nightwatch will run .ts files but I will double check 

Tsv test files are being copied in the post build script using the copy-data-files.js script

Reply:
cool! sorry I missed that :)

Reply:
tried running without a transpile with `ts-node`. Unfortunately nightwatch config still looks for js files only so this wont work